client/settings/test-mode-settings/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Card, CardBody, CheckboxControl } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const TestModeSettings = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+
+	return (
+		<Card className="test-mode-settings">
+			<CardBody size="large">
+				<h4>{ __( 'Test mode', 'woocommerce-payments' ) }</h4>
+				<CheckboxControl
+					checked={ isEnabled }
+					onChange={ setIsEnabled }
+					label={ __(
+						'Enable test mode for payments on the store',
+						'woocommerce-payments'
+					) }
+					help={ interpolateComponents( {
+						mixedString: __(
+							// eslint-disable-next-line max-len
+							"When enabled, you'll be able to test how your customers pay for orders on your store. Use {{testCardHelpLink}}test card numbers{{/testCardHelpLink}} to simulate various transactions. {{learnMoreLink}}Learn more{{/learnMoreLink}}.",
```

Comment:
Maybe split the string into multiple lines?

Reply:
Do you mean split in multiple lines so it's also visually displayed to the merchant in multiple lines? Or just in the code, maybe with a template?
```
`First line...
New line
`
````

I noticed that in other places the string is kept as it is going to be displayed - otherwise I'm afraid that it won't be pulled correctly to translators 👀

Reply:
> Do you mean split in multiple lines so it's also visually displayed to the merchant in multiple lines? Or just in the code, maybe with a template?

Just in the code so you don't have to disable ESLint!

I have just tested this (it sounded like you might be right but I wanted to see how the system works) and it seems like you can actually split the string into multiple lines and it'll work!

Here's how I split the string for testing:

```js
mixedString: __(
	// eslint-disable-next-line max-len
	"When enabled, you'll be able " +
		'to test how your customers pay for orders on your store. Use {{testCardHelpLink}}test card numbers{{/testCardHelpLink}} to simulate various transactions. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
	'woocommerce-payments'
),
```

Then I ran `npm run build:client` and I got this in `languages/woocommerce-payments.pot`:

```
#: client/settings/test-mode-settings/index.js:29
msgid ""
"When enabled, you'll be able to test how your customers pay for orders on "
"your store. Use {{testCardHelpLink}}test card numbers{{/testCardHelpLink}} "
"to simulate various transactions. {{learnMoreLink}}Learn "
"more{{/learnMoreLink}}."
msgstr ""
```

So it seems you can split the string into multiple lines! I'm not saying you should though - I just figured it's better to split the strings instead of silencing the warning.

Reply:
You're right, good idea on also running `build:client`. When using a template, it seems to be generating the string with `\t` characters, which might be undesirable.

Changed to delete the eslint rule suppression 👍